AdminWebsite/AdminWebsite/Models/HealthCheckResponse.cs

```diff
@@ -9,11 +9,13 @@ public HealthCheckResponse()
             BookingsApiHealth = new HealthCheck();
             UserApiHealth = new HealthCheck();
             VideoApiHealth = new HealthCheck();
+            NotificationApiClientHealth = new HealthCheck();
```

Comment:
should this one have "Client" in the name?

Reply:
Fixed